examples/widgets/flutter.yaml

```diff
@@ -1,4 +1,11 @@
 name: widgets
+assets:
+  - assets/world.meta
+  - assets/1.0x/world.png
+  - assets/1.5x/world.png
+  - assets/2.0x/world.png
+  - assets/3.0x/world.png
+  - assets/4.0x/world.png
```

Comment:
five images in an assets/ directory, their five filenames duplicated in a .meta file, and the contents of that file in the flutter.yaml file again... that's a lot of duplication. Can't we auto-derive the .meta and flutter.yaml files from the directory structure at build time?